dockers/docker-base-stretch/dpkg_01_drop

```diff
@@ -20,3 +20,11 @@ path-exclude /usr/share/pyshared/twisted/test*
 path-exclude /usr/lib/python*/dist-packages/twisted/test*
 path-exclude /usr/share/pyshared/twisted/*/test*
 path-exclude /usr/lib/python*/dist-packages/twisted/*/test*
+
+## install the configuration file if it’s currently missing
+force-confmiss
+## combined with confold: overwrite configuration files that you have not modified
+force-confdef
+## do not modify the current configuration file, the new version is installed with a .dpkg-dist suffix
+force-confold
```

Comment:
should be force-connew, always take new configuration since we are not modifying the config in these dockers.

Reply:
When `force-confdef` and `force-confold` combined, it means:
1. old unmodified configuration files will be overwrite  by new ones
2. old modified configuration files will be untouched, and the new version is installed with a .dpkg-dist suffix

---
In reply to: [357311450](https://github.com/Azure/sonic-buildimage/pull/3879#discussion_r357311450) [](ancestors = 357311450)